book/src/lint_configuration.md

```diff
@@ -195,7 +195,7 @@ The maximum cognitive complexity a function can have
 ### disallowed-names
 The list of disallowed names to lint about. NB: `bar` is not here since it has legitimate uses. The value
 `".."` can be used as part of the list to indicate, that the configured values should be appended to the
-default configuration of Clippy. By default any configuration will replace the default value.
+default configuration of Clippy. By default, any configuration will replace the default value.
 
```

Comment:
This file is auto generated, from the doc comments in `clippy_lints::util::conf`. You can update the strings there and then run `cargo collect-metadata` that should update this file and make bors happy :)

Reply:
thanks @xFrednet , fixed.  Storing auto-generated content in git is evil... but alas.